pkg/common/common.go

```diff
@@ -46,8 +48,11 @@ const (
 	DEFAULT_RESYNC_PERIOD = 10 * time.Minute
 
 	// logging verbosity
-	Vuser                    = 1
-	Vadmin                   = 2
-	Vdebug                   = 3
+	Vuser           = 1
+	Vadmin          = 2
+	Vdebug          = 3
 	DEFAULT_VERBOSE = Vuser
+	
+	// the length of the random generated cloning label
+	GENERATED_CLONING_LABEL_LEN = 10
```

Comment:
minor: func `randTmpName` also uses a random name. Maybe this const can be more general and also used there?